src/structures/Invite.js

```diff
@@ -116,11 +116,11 @@ class Invite extends Base {
    * @readonly
    */
   get deletable() {
-    const guild = this.guild;
+    const { guild } = this;
     if (!guild || !this.client.guilds.cache.has(guild.id)) return false;
     if (!guild.me) throw new Error('GUILD_UNCACHED_ME');
```

Comment:
You can do:

```javascript
const { me } = guild;
if (!me) throw new Error('GUILD_UNCACHED_ME');
```

And then in the line 123, do `permissionsFor(me).has` - since `me` is a GuildMember, permissionsFor will return a non-null.

Also in the line 124, `guild.me.permissions` -> `me.permissions`.